.github/workflows/end-to-end.yml

```diff
@@ -110,7 +120,7 @@ jobs:
       env:
         ANSIBLE_CONFIG: ./compliantkubernetes-kubespray/kubespray/ansible.cfg
       with:
-        args: /bin/bash -c "cd compliantkubernetes-kubespray/kubespray && ansible-playbook -i ../../$CK8S_CONFIG_PATH/pipeline-${{ matrix.cluster }}-config/inventory.ini --private-key ../../$CK8S_CONFIG_PATH/id_rsa --become --become-user=root cluster.yml"
+        args: /bin/bash -c "cd compliantkubernetes-kubespray/kubespray && for i in $(seq 1 2); do [ $i -gt 1 ] && sleep 10; ansible-playbook -i ../../$CK8S_CONFIG_PATH/pipeline-${{ matrix.cluster }}-config/inventory.ini --private-key ../../$CK8S_CONFIG_PATH/id_rsa --become --become-user=root cluster.yml && s=0 && break || s=$?; done; (exit $s)"
```

Comment:
This is the retry strategy right? 
Could you add some comments on this step, so we know what the for loop is for?

Reply:
it is an alternative to duplicating the step (e.g what i did for the terrafrom step).
and at least in my tests it worked as expected
i will add some comment

Reply:
Is there a reason to why you chose to not do the duplicate steps here? Because I think that that is a little bit more readable

Reply:
no, not really. I wanted to showcase both options and decide here what is the best one
upvote this comment if you want the step duplicated instead

Reply:
In this case i prefer clear over compact

Reply:
done